helmfile/charts/openstack-monitoring/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.openstackMonitoring.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    {{- include "openstack-monitoring.labels" . | nindent 4}}
+  name: {{ include "openstack-monitoring.fullname" . }}
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    port: http
+    scheme: https
+    tlsConfig:
+      insecureSkipVerify: true
+  jobLabel: component
+  namespaceSelector:
+    matchNames:
+    - kube-system
+  selector:
+    matchLabels:
+      k8s-app: openstack-cloud-controller-manager
```

Comment:
NIT but I'm guessing this could also be changed to use the helpers similar to what @Xartos suggested earlier for labels etc? :smile:

Reply:
Yeah i asked @Xartos that myself thou i should probably have done it here in the comments. But his answer was to have them established in the yaml file and not change them to the helper.